flask_moment.py

```diff
@@ -112,32 +119,37 @@ def _timestamp_as_iso_8601(self, timestamp):
             tz = 'Z'
         return timestamp.strftime('%Y-%m-%dT%H:%M:%S' + tz)
 
-    def _render(self, format, refresh=False):
+    def _render(self, format, refresh=False, **kwargs):
         t = self._timestamp_as_iso_8601(self.timestamp)
-        return Markup(('<span class="flask-moment" data-timestamp="%s" ' +
-                       'data-format="%s" data-refresh="%d" ' +
-                       'style="display: none">%s</span>') %
-                      (t, format, int(refresh) * 60000, t))
+        opener = (f'<span class="flask-moment" data-timestamp="{t}" '
+                  f'data-format="{format}" '
+                  f'data-refresh="{int(refresh) * 60000}" '
+                  f'style="display: none"')
```

Comment:
Python 2.7 and 3.5 are still supported, so can't use f-strings unfortuantely.

Reply:
I can convert those easy enough (didn't know about the backward compatibility issue of f-strings!) but I'm wondering if this is the right way to go about the conversion anyway. What do you think?